static/app/components/replays/diff/replayDiffChooser.tsx

```diff
@@ -0,0 +1,75 @@
+import Alert from 'sentry/components/alert';
+import {Flex} from 'sentry/components/container/flex';
+import {ReplaySideBySideImageDiff} from 'sentry/components/replays/diff/replaySideBySideImageDiff';
+import {ReplaySliderDiff} from 'sentry/components/replays/diff/replaySliderDiff';
+import {ReplayTextDiff} from 'sentry/components/replays/diff/replayTextDiff';
+import {TabList, TabPanels, Tabs} from 'sentry/components/tabs';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type ReplayReader from 'sentry/utils/replays/replayReader';
+
+interface Props {
+  leftOffsetMs: number;
+  replay: null | ReplayReader;
+  rightOffsetMs: number;
+  defaultTab?: DiffType;
+}
+
+export enum DiffType {
```

Comment:
just still recommending const enums since they work like strings instead of a weird object
```suggestion
export const enum DiffType {
```